CHANGELOG.md

```diff
@@ -19,9 +19,9 @@ For official release notes for Docker Engine CE and Docker Engine EE, visit the
 
 ### Runtime
 
-* Bump Golang 1.12.15. [docker/engine#452](https://github.com/docker/engine/pull/452)
-* Update containerd binary to v1.2.11. [docker/engine#428](https://github.com/docker/engine/pull/428)
-* Update to runc v1.0.0-rc9. [docker/engine#428](https://github.com/docker/engine/pull/428)
+* Bump Golang 1.12.16. [moby/moby#40433](https://github.com/moby/moby/pull/40433)
+* Update containerd binary to v1.2.12. [moby/moby#40433](https://github.com/moby/moby/pull/40453)
+* Update to runc v1.0.0-rc10. [moby/moby#40433](https://github.com/moby/moby/pull/40453)
 - Fix possible runtime panic in Lgetxattr. [docker/engine#454](https://github.com/docker/engine/pull/454)
```

Comment:
Should we mention the CVE's that were fixed both in Golang and Runc/containerd ?

Reply:
From a quick scan of the CHANGELOG, I don't think we've specified Golang CVE's in the past, the links should have the CVE details in there